### ISSUE TITLE
Datatable not displaying properly with Read the Docs theme

### ISSUE THREAD

**Issue by vasib** [useblocks/sphinx-needs#771]
I'm using Read the Docs theme 1.0.0/1.1.0 and this is how a `needstable` with `datatables` style looks like:

![image](https://user-images.githubusercontent.com/90906984/199627043-30b59a7d-77b0-4593-a066-2bd291732dcd.png)

If I remove the `overflow: auto` css property from `.wy-table-responsive` it fixes the issue, but I'm not sure if that is the right way to do this. Since this is very popular theme I was assuming it would work out of the box.
The regular `table` style displays fine.

Is this a *needs* issue or a *rtd* one?

**Comment by twodrops** [comment]
@vasib This is a known issue and there is a PR created at RTD by @danwos before several months for this
https://github.com/readthedocs/sphinx_rtd_theme/pull/1218
The RTD guys are too slow on their PRs.

See issues
https://github.com/readthedocs/sphinx_rtd_theme/issues/1179
https://github.com/readthedocs/sphinx_rtd_theme/issues/767

**Comment by vasib** [comment]
Thanks for the info. That 1 line PR has been sitting there for more than a year 😐.

**Comment by twodrops** [comment]
@vasib 
That's right. We, at our project, are seriously thinking of moving to immaterial theme https://github.com/jbms/sphinx-immaterial

That number doesn't speak well for RTD Theme anymore
![image](https://user-images.githubusercontent.com/103930/200197697-8422428e-22ad-4977-bf8b-28257ccfdb31.png)

**Comment by chrisjsewell** [comment]
#1920 (merged, in the next release) removes half of this collision: DataTables and its `div.dataTables_wrapper` are gone, and the interactive table now sits in a scroll frame of its own. Under sphinx_rtd_theme the theme's own `div.wy-table-responsive` wrapper — the one whose `overflow: auto` this issue is about — ends up nested *inside* that frame, and in our docs build (which renders against sphinx_rtd_theme as one of five themes) a 900 px table scrolls correctly inside it, with the widget's controls staying put.

What we have not verified is a plain project on that theme without our docs' own small override stylesheet, so this is "probably fixed" rather than "fixed". If you still have the project from the screenshot above, could you try it against the next release (or `pip install git+https://github.com/useblocks/sphinx-needs@master#subdirectory=packages/sphinx-needs`) and say whether the table still escapes its container? If it does, the remaining rule belongs in the package rather than in our docs, and that is a small change.
